src/components/Timeline/EventFooter.js

```diff
@@ -32,4 +34,8 @@ const EventFooter = ({pagination, dispatch}) => {
   )
 }
 
-export default EventFooter
+export const mapStateToProps = (state, ownProps) => ({
+  pagination: state.events.pages
+})
+
+export default connect(mapStateToProps)(EventFooter)
```

Comment:
why do you add mapStateToProps to EventFooter? Where does the pagination value come from previously?

Reply:
It was previously passed down from Timeline, but we should take things out of state as close to where they're being used as possible rather than passing them around after taking them out of state. Doing so avoids unnecessary re-rendering within React.